awx/main/access.py

```diff
@@ -588,17 +588,44 @@ def can_delete(self, obj):
 
 
 class InstanceGroupAccess(BaseAccess):
+    """
+    I can see Instance Groups when I am:
+       - a superuser(system administrator)
+       - an organization admin to which an instance group is associated
+       - at least read_role on the instance group
+    I can edit Instance Groups when I am:
+       - a superuser
+       - admin role on the Instance group
+    I can add/delete Instance Groups:
+       - a superuser(system administrator)
```

Comment:
I'm not sure why we should continue the special superuser criteria for deletion after the `admin_role` has been introduced. Certainly, the special IGs like controlplane must be protected, but absent that case, we should defer to the standard logic, which would defer to `can_admin`. I would prefer to see a call to `super()` in this case.

Reply:
I take it back, creation should be congruent with deletion (not admin).